Encounter1/LocationTable.cs

```diff
@@ -12,9 +12,12 @@
 
```

Comment:
Since you changed class name, I believe file name should also be changed to match it